build.ps1

```diff
@@ -5,11 +5,14 @@
 ##########################################################################
 
 <#
+
```

Comment:
I pulled this file directly from Cake but I did need to add one line to make it work with Cake as a tool. When we finally drop support for .NET Standard 1.5 we can just use the file from Cake directly. https://github.com/durran/mongo-csharp-driver/pull/2/files#diff-3258bc4162690adead72c70b672093e68c92e86a4628148f0015d61738f21b5aR255

Reply:
Have you used this file: https://github.com/cake-build/resources/blob/master/dotnet-framework/build.ps1?
If so, I see some more small changes in the original file, they don't look too serious, is there a reason why they missed?

Reply:
The diff that was highlighted I'm not sure we needed. We define the $Script value as build.cake on line 40, so I don't know why we would need to check if it exists later and set it. If we simply want the exact same one as the link you sent I can use it verbatim, since it's slightly different.

Reply:
I pulled this file directly from the Cake repo - the only thing I modified is where this comment is. https://github.com/durran/mongo-csharp-driver/pull/2/files/a9f97448fe0aee826bed2a3ca9f443807e04c978#r516939866

Reply:
OK. But:

> how will we make sure such modifications are preserved in the future when someone overwrites this file with a newer version from the Cake repo?